### PR TITLE
feature/AT-SSO-BE 17 Initial Post Implementation

### PR DESCRIPTION
**Pull request by carla-paz**
Initial POST Implementation - No validation

### CODE REVIEW

File: src/main/java/com/resources/model/User.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
+package com.resources.model;
```
Comment (at-internship) [inline src/main/java/com/resources/model/User.java]: could be this name package com.agilethought.internship.sso.model

File: src/main/java/com/resources/services/ServiceApplicationimpl.java [hunk 2]
```diff
@@ -0,0 +1,27 @@
+package com.resources.services;
```
Comment (at-internship) [inline src/main/java/com/resources/services/ServiceApplicationimpl.java]: could be this package name com.agilethought.internship.sso.services

File: src/main/java/com/resources/services/ServiceApplication.java [hunk 3]
```diff
@@ -0,0 +1,10 @@
+package com.resources.services;
```
Comment (at-internship) [inline src/main/java/com/resources/services/ServiceApplication.java]: could be this package name com.agilethought.internship.sso.services

File: src/main/java/com/resources/repository/RepositoryApplication.java [hunk 4]
```diff
@@ -0,0 +1,10 @@
+package com.resources.repository;
```
Comment (at-internship) [inline src/main/java/com/resources/repository/RepositoryApplication.java]: could be this package name com.agilethought.internship.sso.repository

File: src/main/java/com/resources/model/UserId.java [hunk 5]
```diff
@@ -0,0 +1,10 @@
+package com.resources.model;
```
Comment (at-internship) [inline src/main/java/com/resources/model/UserId.java]: could be this package name com.agilethought.internship.sso.model

File: src/main/java/com/resources/controller/ControllerAplication.java [hunk 6]
```diff
@@ -1,16 +1,50 @@
...
+		MappingMongoConverter converter = new MappingMongoConverter(dbRefResolver, mongoMappingContext);
+		converter.setTypeMapper(new DefaultMongoTypeMapper(null));
+		return converter;
+	}
+	
+	@PostMapping(value = "/at-sso-api/api/v1/user", produces = "application/json")
+	@ResponseStatus(value = HttpStatus.CREATED)
+	public UserId createUser(@RequestBody User user) {
+		UserId userId = serviceApplication.createUser(user);
+		return userId;
```
Comment (at-internship) [inline src/main/java/com/resources/controller/ControllerAplication.java]: just be sure that we returning the right Response:

You can use `return new ResponseEntity<>(_userObject, HttpStatus.CREATED);`

File: src/main/java/com/resources/controller/ControllerAplication.java [hunk 7]
```diff
@@ -1,16 +1,50 @@
 package com.resources.controller;
```
Comment (at-internship) [inline src/main/java/com/resources/controller/ControllerAplication.java]: could be this name package com.agilethought.internship.sso.controller

File: src/main/java/com/resources/DemoApplication.java [hunk 8]
```diff
@@ -2,8 +2,9 @@
 
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.boot.autoconfigure.jdbc.DataSourceAutoConfiguration;
 
-@SpringBootApplication
+@SpringBootApplication(exclude = {DataSourceAutoConfiguration.class})
 public class DemoApplication {
```
Comment (at-internship) [inline src/main/java/com/resources/DemoApplication.java]: Change this class name to ATSSOApplication

File: src/main/java/com/resources/services/ServiceApplicationimpl.java [hunk 9]
```diff
@@ -0,0 +1,27 @@
...
+@Service
+@Slf4j
+public class ServiceApplicationimpl implements ServiceApplication {
+
+	@Autowired
+	private RepositoryApplication repositoryApplication;
+	
+	@Override
+	public UserId createUser(User user) {
+		String userIdDb = repositoryApplication.save(user).getId();
```
Comment (at-internship) [inline src/main/java/com/resources/services/ServiceApplicationimpl.java]: You must return User _user

**Review by at-internship — CHANGES_REQUESTED** [review]

**Review by at-internship — CHANGES_REQUESTED** [review]
Please take a look

**Review by VictorMunguiaEnciso — APPROVED** [review]

**Review by rodrigo-velasco — APPROVED** [review]

**Pull request merged**